common/tools/dev-tool/src/config/rollup.base.config.ts

```diff
@@ -125,7 +125,7 @@ export function makeBrowserTestConfig(): RollupOptions {
           // Chai's strange internal architecture makes it impossible to statically
           // analyze its exports.
           chai: ["version", "use", "util", "config", "expect", "should", "assert"],
-          ...openTelemetryCommonJs()
+          events: ["EventEmitter"]
```

Comment:
I think the definition of `openTelemetryCommonJs` should be deleted too if it is no longer needed.

I wonder who uses `events`.

Reply:
`events` started out in the template package and my guess is that it got copied around :)

Reply:
We're close to being able to remove this crud - I just know that every rollup file imports this method and uses it, so rather than increasing the PR size I opted to leave the definiton. But I will log a separate issue for me to remove it (here and everywhere that has it)